healvis/observatory.py

```diff
@@ -269,7 +311,7 @@ def make_visibilities(self, shell, Nprocs=1, beam_pol='pI'):
         Nfin = mp.Value('i', 0)
 
         if Nprocs > 1 and not isinstance(shell.data, mparray):
-            warnings.warn("Caution: SkyModel data array is not in shared memory. With Nprocs > 1, this will cause duplication")
+            warnings.warn("Caution: SkyModel data array is not in shared memory. With Nprocs > 1, this will cause duplication.")
```

Comment:
is there an easy way to make it in shared memory? Is that documented somewhere?

Reply:
It's not well documented, but the shared memory class is in utils.py as `mparray`. Reading in the sky model using `SkyModel.read_hdf5`, there is an option to read the data into a shared memory array. This is the default behavior of `simulator.run_simulation`.